src/__test__/example.test.js

```diff
@@ -0,0 +1,13 @@
+describe('this is for group test examples', () => {
+  it('description', () => {
+    expect(true).toBe(true);
+  });
+
+  it('description 2', () => {
+    expect(true).toBe(true);
+  });
+
+  it('description 3', () => {
+    expect(true).toBe(true);
+  });
+});
```

Comment:
This examples seem to be a good starting point, in several projects I've seem the tests defined like this